src/platforms/esp32/test/main/test_avm_otp21.c

```diff
@@ -0,0 +1,219 @@
+const unsigned char test_avm_otp21[1732] = {
```

Comment:
Could you leave a comment that says which source files are used to generate the AVM data in this module?  It's difficult to decipher from the code, even though it is noted in the commit comments (not the first place I would go to look).

Reply:
ok, I will a comment above the `#include`. That file is automatically generated, hence I would loose any manual change to it.